tests/unit/test_service_plan.py

```diff
@@ -0,0 +1,79 @@
+import json
+
+import pytest
+from migrator.extensions import get_cf_client
+from migrator import service_plan
+import requests_mock
+
+
+def get_test_client(fake_requests):
```

Comment:
If more tests will need to authenticate, maybe move this into a fixture or some type of test helper for more portability.